### PR TITLE
[incubator/chartmuseum] update documentation

### PR DESCRIPTION
**Pull request by stefsuse**
Update documentation to help move this chart into stable.

*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

File: incubator/chartmuseum/README.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
 # ChartMuseum Helm Chart
 
-Work in progress...
+Deploy your own private ChartMuseum.   
 
-Please see https://github.com/chartmuseum/chartmuseum
\ No newline at end of file
+Please also see https://github.com/chartmuseum/chartmuseum
```
Comment (jdolitsky) [inline incubator/chartmuseum/README.md]: Should be switched to https://github.com/kubernetes-helm/chartmuseum

File: incubator/chartmuseum/README.md [hunk 2]
```diff
@@ -1,5 +1,78 @@
...
\ No newline at end of file
+Please also see https://github.com/chartmuseum/chartmuseum   
+
+## Prerequisites
+
+* Kubernetes with extensions/v1beta1 available
+* [If enabled] A persistent storage resource and RW access to it
+* [If enabled] Kubernetes StorageClass for dynamic provisioning
+
+## Configuration
```
Comment (jdolitsky) [inline incubator/chartmuseum/README.md:13]: Can we add a comment here to say "check out values.yaml for all the possible configuration options" ?

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: incubator/chartmuseum/README.md [hunk 3]
```diff
@@ -1,5 +1,78 @@
...
+
+By default this chart will not have persistent storage.   
+
+For a more robust solution supply helm install with a custom values.yaml   
+You are also required to create the StorageClass resource ahead of time:   
+```
+kubectl create -f /path/to/storage_class.yaml
+```
+
+The following tables lists the configurable parameters of the vault chart and
```
Comment (jdolitsky) [inline incubator/chartmuseum/README.md]: "vault" chart?

File: incubator/chartmuseum/README.md [hunk 4]
```diff
@@ -1,5 +1,78 @@
...
+| `resources.requests.memory`  | Container requested memory                  | `64Mi`                                              |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to
+`helm install`.
+
+
+## Installation
+
+```shell
+helm install --name my-chartmuseum -f values.yaml incubator/chartmuseum
```
Comment (jdolitsky) [inline incubator/chartmuseum/README.md]: This should probably be `-f custom.yaml` or something similar. `values.yaml` should contain defaults then be overridden by another yaml

File: incubator/chartmuseum/README.md [hunk 5]
```diff
@@ -1,5 +1,78 @@
...
+```
+
+## Uninstall 
+
+By default, a deliberate uninstall will result in the persistent volume 
+claim being deleted.   
+
+```shell
+helm delete my-chartmuseum
```
Comment (jdolitsky) [inline incubator/chartmuseum/README.md:57]: Maybe want to add a `--purge` to the end of this

File: incubator/chartmuseum/README.md (hunk 3 above)
Comment (stefsuse) [reply incubator/chartmuseum/README.md]: fixed

**Pull request merged**